mock/mockimpl/kvproc/subdoc.go

```diff
@@ -49,7 +51,7 @@ func subdocReorder(ops []*SubDocOp) *subdocOpList {
 	}
 }
 
-func (e *Engine) executeSdOps(doc, newMeta *mockdb.Document, ops []*SubDocOp) ([]*SubDocResult, error) {
+func (e *Engine) executeSdOps(doc, newMeta *mockdb.Document, ops []*SubDocOp, continueOnOpError bool) ([]*SubDocResult, error) {
```

Comment:
I feel like we should imply this bool from the ops somehow but that's just being picky.